cmd/substreams-sink-postgres/generate_csv.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/streamingfast/cli"
+	. "github.com/streamingfast/cli"
+	"github.com/streamingfast/cli/sflags"
+	"github.com/streamingfast/shutter"
+	sink "github.com/streamingfast/substreams-sink"
+	"github.com/streamingfast/substreams-sink-postgres/db"
+	"github.com/streamingfast/substreams-sink-postgres/sinker"
+	"go.uber.org/zap"
+)
+
+var generateCsvCmd = Command(generateCsvE,
+	"generate-csv <psql_dsn> <endpoint> <manifest> <module> [<start>:<stop>]",
+	"Runs first load sink process",
```

Comment:
Let's improve the description a bit, talking about file sinking + file generation. Let's also add a `Description` that would explain in richer details what is this command about and "how" to use it.
```suggestion
	"Runs first load sink process",
	Description(`
	   Why this command exist and what it does.
	   
	   How to use it .. (1-2 paragraph) you decide a range, you decide a folder, can be a remove store, see `dstore`. Talk about cursors and how they are dealt with.
	`),
```

Reply:
Done